src/index.css

```diff
@@ -1,13 +1,153 @@
 :root {
```

Comment:
Could the CSS file be divided in several parts connected to each component for easier access and readability?